### ISSUE TITLE
Send message to iphone via apple messages

### ISSUE THREAD

**Issue by spinkney** [ahertel/wholefoods-delivery-slot-finder#7]
Is it possible to set the script so that it imessages an iphone from the messages app when the slot becomes available?

**Comment by spinkney** [comment]
Okay, I found how you can do this.

Here it is below
```applescript
...
else if siteText contains slot_page_keyword and (siteText contains "AM" or siteText contains "PM") then
		-- landed on delivery slot page and delivery slot selection drop down appears aka. slot found!
		tell application "Messages"
			
			set iMessageid to get id of first service
			
			set theNumber to buddy "##########" of service id iMessageid --enter the number that you want to send your message to here
			
			send "Get to amazon.com now, slots are open!" to theNumber
			
		end tell
		
		display notification "Found delivery slot!" with title "Amazon" sound name "Sosumi"
		say "Success: Delivery slot found"
		set found_slot to true
...
```

**Comment by ahertel** [comment]
this is a great idea for those who can’t be around their computers all the time!
 I’d love to integrate this into the main script while crediting you. You could submit a pull request or I can just copy/paste myself and credit you with a comment in the script. Let me know what you prefer. Thanks!

**Comment by bsteinlo** [comment]
> Okay, I found how you can do this.
> 
> Here it is below
> 
> ```applescript
> ...
> else if siteText contains slot_page_keyword and (siteText contains "AM" or siteText contains "PM") then
> 		-- landed on delivery slot page and delivery slot selection drop down appears aka. slot found!
> 		tell application "Messages"
> 			
> 			set iMessageid to get id of first service
> 			
> 			set theNumber to buddy "##########" of service id iMessageid --enter the number that you want to send your message to here
> 			
> 			send "Get to amazon.com now, slots are open!" to theNumber
> 			
> 		end tell
> 		
> 		display notification "Found delivery slot!" with title "Amazon" sound name "Sosumi"
> 		say "Success: Delivery slot found"
> 		set found_slot to true
> ...
> ```

I duplicated the script and tried adding this but the formatting is all off, I feel like I am missing something silly.

**Comment by marvinwh2020** [comment]
> Okay, I found how you can do this.
> 
> Here it is below
> 
> ```applescript
> ...
> else if siteText contains slot_page_keyword and (siteText contains "AM" or siteText contains "PM") then
> 		-- landed on delivery slot page and delivery slot selection drop down appears aka. slot found!
> 		tell application "Messages"
> 			
> 			set iMessageid to get id of first service
> 			
> 			set theNumber to buddy "##########" of service id iMessageid --enter the number that you want to send your message to here
> 			
> 			send "Get to amazon.com now, slots are open!" to theNumber
> 			
> 		end tell
> 		
> 		display notification "Found delivery slot!" with title "Amazon" sound name "Sosumi"
> 		say "Success: Delivery slot found"
> 		set found_slot to true
> ...
> ```

Can you please tell me where to put this in the original code. I have no experience with this. Thanks marvin

**Comment by prateekm21** [comment]
You can put imessage slot here for found
https://github.com/ahertel/Amazon-Fresh-Whole-Foods-delivery-slot-finder/blob/master/delivery-window-finder.applescript#L104

and to test you can put where it prints logs  "no slots available" - [link](https://github.com/ahertel/Amazon-Fresh-Whole-Foods-delivery-slot-finder/blob/master/delivery-window-finder.applescript#L74)
you will get text for not available; once works you can remove

**Comment by spinkney** [comment]
@ahertel, feel free to add it in. You can just say thanks to spinkney for providing imesage support in a comment above the code. Thanks!!

**Comment by ahertel** [comment]
@spinkney thanks for getting back to me. I've added it with a credit to you, and I wrapped it in user interface to ensure data validity and an simpler user experience.
If you happen to try it, I'd love to hear your thoughts.

@prateekm21, I replied to your pull request but wanted to thank you again here for your push to get this integrated.

@marvinwh2020 @bsteinlo hope can try the new text message feature. let me know if it works for you.
